### PR TITLE
Updated French translations

### PR DESCRIPTION
**Pull request by tmlmt**
This PR adds translations for all missing strings in French as of June 5 and corrects multiple existing translations in order to improve coherence and use better idioms. 

Example of corrected incoherence (settled with the one in bold):
- Use of multiple words for "Plan": **abonnement**, forfait, plan
- Use of multiple words for "Sign in": **se connecter**, s'inscrire (which is actually a mistranslation). 
- Different translations for "Welcome back" between `ghost.json` and `portal.json`: **Bon retour parmi nous**, Bienvenue

Example of idiomatic improvements, also better fitting with typical uses by other online services:
- You wouldn't say "Bonne chance" (litteraly "good luck") as a light greeting at the end of an email. 
- "Il y a eu un problème avec" --> "Une erreur est survenue"
- "Forever" --> "Permanent" instead of "Pour toujours" (you would fall in love _pour toujours_, but would rather sign up to something "Permanently")
- "S'il vous plait, mettez" --> "Veuillez"

An additional comment for: 
`"{{amount}} off for first {{period}}.": "A label for an offer, explaining how long the discount is for (either first month or year)"` 
is that `first year` would translate in `la première année` while `first month` in `le premier mois`. This implies that the combined translation can look like "{{amount}} de réduction pour le/la premier/ère {{period}}." but it's rather ugly. Could you consider splitting this string into 3? 
- `{{amount}} off for {{period}}`
- `first month`
- `first year`

Hope it helps :) and I'm of course open to discussions and improvements.

### CODE REVIEW

**Comment by thibaultmalfoy** [comment]
Excellent work! I agree with you about "{{amount}} off for first {{period}}."

**Review by sagzy — COMMENTED** [review]
Great work! Thanks for the updates and new additions. :) 

Looking good to me, suggested one change @tmlmt

File: ghost/i18n/locales/fr/ghost.json
Comment (tmlmt) [reply ghost/i18n/locales/fr/ghost.json:5]: Hey @sagzy thanks for commenting. I understood from the context file that these strings where to be used in email subject titles. It's a matter of convention I guess, one could use e.g. either "Completion of (your) sign-up" (as in "an action is required for the confirmation of your signup" or "Confirm signup" in the imperative form. I took the liberty to suggest the first one, but can understand that you have chosen the imperative form in the original English version so we could make it coherent in all languages. Shall I edit the PR? :)

I also suggest that the context file is updated to reflect that the imperative form is expected here.

Comment (sagzy) [reply ghost/i18n/locales/fr/ghost.json:5]: Yes, I can def. follow your reasoning – here, I think the imperative form works better as there is a call to action in the body to complete the signup / confirm the email. Yes please go ahead with the changes, and also be free to add an additional note in the context file. Thanks!

**Comment by sagzy** [comment]
@daniellockyer regarding `{{amount}} off for first {{period}}` (cf. @tmlmt comment in the PR description): I agree, it would make sense to split it for any language that has a different masculine/feminine pronoun for month vs year (French, German, probably more). Is it best to do it now, or wait until the other translations PRs are merged first?

File: ghost/i18n/locales/fr/ghost.json
Comment (tmlmt) [reply ghost/i18n/locales/fr/ghost.json]: Done :)

**Comment by tmlmt** [comment]
@sagzy along with addressing your review comment, I've also improved one of the translations in f08d4c0, a good idea which was in the other PR #16948 by @jeremielondon (which I have credited :) for the inspo). It brings the translation closer to its English original version.

**Review by sagzy — APPROVED** [review]
@tmlmt sorry for the delay, totally forgot to hit the approve button last week! Approved :)

@daniellockyer good to merge 👍 Also cf. this [comment](https://github.com/TryGhost/Ghost/pull/16942#issuecomment-1580545594) for a future PR

**Comment by daniellockyer** [comment]
Thanks @sagzy and @tmlmt! 🕺🏻

**Comment by alainsameja** [comment]
> This PR adds translations for all missing strings in French as of June 5 and corrects multiple existing translations in order to improve coherence and use better idioms.
> 
> Example of corrected incoherence (settled with the one in bold):
> 
>     * Use of multiple words for "Plan": **abonnement**, forfait, plan
> 
>     * Use of multiple words for "Sign in": **se connecter**, s'inscrire (which is actually a mistranslation).
> 
>     * Different translations for "Welcome back" between `ghost.json` and `portal.json`: **Bon retour parmi nous**, Bienvenue
> 
> 
> Example of idiomatic improvements, also better fitting with typical uses by other online services:
> 
>     * You wouldn't say "Bonne chance" (litteraly "good luck") as a light greeting at the end of an email.
> 
>     * "Il y a eu un problème avec" --> "Une erreur est survenue"
> 
>     * "Forever" --> "Permanent" instead of "Pour toujours" (you would fall in love _pour toujours_, but would rather sign up to something "Permanently")
> 
>     * "S'il vous plait, mettez" --> "Veuillez"
> 
> 
> An additional comment for: `"{{amount}} off for first {{period}}.": "A label for an offer, explaining how long the discount is for (either first month or year)"` is that `first year` would translate in `la première année` while `first month` in `le premier mois`. This implies that the combined translation can look like "{{amount}} de réduction pour le/la premier/ère {{period}}." but it's rather ugly. Could you consider splitting this string into 3?
> 
>     * `{{amount}} off for {{period}}`
> 
>     * `first month`
> 
>     * `first year`
> 
> 
> Hope it helps :) and I'm of course open to discussions and improvements.



> This PR adds translations for all missing strings in French as of June 5 and corrects multiple existing translations in order to improve coherence and use better idioms.
> 
> Example of corrected incoherence (settled with the one in bold):
> 
>     * Use of multiple words for "Plan": **abonnement**, forfait, plan
> 
>     * Use of multiple words for "Sign in": **se connecter**, s'inscrire (which is actually a mistranslation).
> 
>     * Different translations for "Welcome back" between `ghost.json` and `portal.json`: **Bon retour parmi nous**, Bienvenue
> 
> 
> Example of idiomatic improvements, also better fitting with typical uses by other online services:
> 
>     * You wouldn't say "Bonne chance" (litteraly "good luck") as a light greeting at the end of an email.
> 
>     * "Il y a eu un problème avec" --> "Une erreur est survenue"
> 
>     * "Forever" --> "Permanent" instead of "Pour toujours" (you would fall in love _pour toujours_, but would rather sign up to something "Permanently")
> 
>     * "S'il vous plait, mettez" --> "Veuillez"
> 
> 
> An additional comment for: `"{{amount}} off for first {{period}}.": "A label for an offer, explaining how long the discount is for (either first month or year)"` is that `first year` would translate in `la première année` while `first month` in `le premier mois`. This implies that the combined translation can look like "{{amount}} de réduction pour le/la premier/ère {{period}}." but it's rather ugly. Could you consider splitting this string into 3?
> 
>     * `{{amount}} off for {{period}}`
> 
>     * `first month`
> 
>     * `first year`
> 
> 
> Hope it helps :) and I'm of course open to discussions and improvements.

the translation of "Welcome back to" as "Bienvenue de retour chez ", is not in correct french language. a better suggestion should be " Bienvenue. Nous sommes heureux de vous revoir chez ".

Thanks

**Pull request merged**
